frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/default-template.ts

```diff
@@ -55,6 +61,18 @@ export const resolveDefaultVMTemplate = (params: DefaultVMLikeEntityParams): Tem
         .asResource(),
     });
 
+  const cloudInitVolume = vm.getCloudInitVolume();
+  const cloudInitHelper = new CloudInitDataHelper(cloudInitVolume?.cloudInitNoCloud);
+  if (!cloudInitHelper.isEmpty() && cloudInitHelper.hasKey(CloudInitDataFormKeys.PASSWORD)) {
+    cloudInitHelper.set(CloudInitDataFormKeys.PASSWORD, generateCloudInitPassword());
+    vm.updateVolume(
+      new VolumeWrapper()
```

Comment:
we should use the original volume and not create a new one

Reply:
fixed @suomiy